apis/v1/routes/web.api.js

```diff
@@ -1,6 +1,56 @@
-const router = require("express").Router();
+// ---------------
+// Based Routes
+// ---------------
+const express = require("express");
+const {
+    getUserById,
+    updateUser,
+    deleteUser,
+    
+    // Temporary routes - for testing (will be removed)
+    getAllUsers,
+    createUser,
+    loginUser
 
+} = require("../controllers/user.controller");
 
+const {
+    createAddress,
+    getAddressById,
+    updateAddress,
+    softDeleteAddress,
+    destroyAddress,
+} = require("../controllers/addresses.controller");
+
+// ---------------
+
+
+// ---------------
+// Based Variatbles
+// ---------------
+// const router = require("express").Router();
+const router = express.Router();
+// ---------------
+
+
+// ---------------
+// Based Routers
+// ---------------
+router.route('/users/:user_id').get(getUserById).put(updateUser).delete(deleteUser);
+router.route('/users/:id/addresses').get(getAddressById).post(createAddress).put(updateAddress).delete(softDeleteAddress);
```

Comment:
This route should have a way to identify the user as well as the address to manipulate. 
We need to know which user wants to access this route, as well as what address the user wants to manipulate. 
I suggest splitting this route.
router.route("/users/:user_id/addresses") for creating and getting all user addresses.
router.route("/users/:user_id/addresses/:address_id") for getting a single address updating and deleting.